### PR TITLE
Style landing page

### PR DESCRIPTION
**Pull request by ceemion**
@SundayAdefila Please review.... WARNING!!! It may take a while because there are lots of file changes. 😛

### CODE REVIEW

File: app/assets/javascripts/angular/templates/home.html [hunk 1]
```diff
@@ -69,24 +76,32 @@
...
         <!-- Discover Box -->
         <div id="cDiscover" class="c-discover">
-          <div class="row columns text-center">
+          <div class="row columns text-center discover-box">
             <h3>Discover</h3>
             <hr>
-            <p>find the right career path for you</p>
-            <div class="content"></div>
+            <p class="headline">find the right career path for you</p>
+            <div class="content">
```
Comment (SundayAdefila) [inline app/assets/javascripts/angular/templates/home.html]: did you really expect me to  read this? the layout is distracting.

File: app/assets/stylesheets/_settings.scss [hunk 2]
```diff
@@ -52,7 +52,7 @@ $foundation-palette: (
   primary: #09798e,
   secondary: #828282,
   success: #40be5c,
-  warning: #ff9d06,
+  warning: #ff8c1e, //ff9d06
```
Comment (SundayAdefila) [inline app/assets/stylesheets/_settings.scss]: do you plan to use the commented out hex color?

File: app/assets/javascripts/global.js [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
-  // Some custom JavaScript here.
+
+  // Toggle header slider images
+  var toggleClasses = function (divId, classesArray) {
+    $(divId).each(function () {
+      this.className = classesArray[($.inArray(this.className, classesArray) + 1) % classesArray.length];
+    });
+  };
+
+  window.setInterval(function () { toggleClasses ('#cg_slider', ['one', 'two', 'three']) }, 5000);
```
Comment (SundayAdefila) [inline app/assets/javascripts/global.js]: what bout if there more than 3 classes(3 pics?)

File: app/assets/stylesheets/home/_header.scss [hunk 4]
```diff
@@ -1,14 +1,78 @@
...
+      background: image_url('home/slider/img11.jpg') center center no-repeat;
+      background-size: cover;
+      -webkit-transition: all 2.5s ease-in-out;
+      -moz-transition: all 2.5s ease-in-out;
+      -ms-transition: all 2.5s ease-in-out;
+      -o-transition: all 2.5s ease-in-out;
+      transition: all 2.5s ease-in-out;
+    }
+
+    &.two {
```
Comment (SundayAdefila) [inline app/assets/stylesheets/home/_header.scss]: what does "&." means? education time!

**Comment by SundayAdefila** [comment]
I've left my first batch of comments... ;)

**Comment by SundayAdefila** [comment]
I've left my first batch of comments... ;).. and then mistakenly closed... 🙈

File: app/assets/javascripts/angular/templates/home.html (hunk 1 above)
Comment (ceemion) [inline app/assets/javascripts/angular/templates/home.html]: Lool. You should read it. It's just placeholder text. I'm thinking of replacing it with a chart of the flow of a seeker who needs to discover a career path and how CGuides helps to achieve it.

File: app/assets/javascripts/global.js (hunk 3 above)
Comment (ceemion) [inline app/assets/javascripts/global.js]: We add the third class in the array `['one', 'two', 'three']` and define it in the CSS.

File: app/assets/stylesheets/_settings.scss (hunk 2 above)
Comment (ceemion) [inline app/assets/stylesheets/_settings.scss]: It's the default foundation colour. I may, but wanna tryout our colour first to see how it flies.

File: app/assets/javascripts/global.js (hunk 3 above)
Comment (SundayAdefila) [inline app/assets/javascripts/global.js]: so, it's not really dynamic again? /giphy confused...

File: app/assets/stylesheets/home/_header.scss (hunk 4 above)
Comment (ceemion) [inline app/assets/stylesheets/home/_header.scss]: `&.` is SASS. It's used to place a nested class on the same level as the parent.

`#cg_slider {
  &.one {
  }
}
`

is same as 
`
#cg_slider.one {
}
`

File: app/assets/javascripts/global.js (hunk 3 above)
Comment (ceemion) [inline app/assets/javascripts/global.js]: It is na. toggleClasses function takes any length of array. What changes is what you pass into it. Got it now?

**Pull request merged**